tech_grants_pipeline.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+export PYTHONPATH='.'
```

Comment:
why that?

Reply:
I was having problems running the bash script without including it (it complained it didn't know where nutrition_labels.grant_tagger.py was). But I do often have this problem for this project (not sure why) so maybe its just needed on my computer